base_layer/core/src/transaction.rs

```diff
@@ -214,7 +256,7 @@ impl Hashable for TransactionInput {
 /// overflow and the ownership of the private key.
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 pub struct TransactionOutput {
-    /// Options for an output's structure or use
+    /// Options for an output's structure or use, coinbase transaction will be checked for maturity lenght
```

Comment:
nit: typo.

Coinbase outputs have maturity, in general I suggest simply calling this a lock height

Reply:
Maturity is for me more clear what it is, and I recommend sticking to so that it becomes clear what it is. All thou on normal terms only coinbase utxo have maturity, our code allows all utxo's to have it. It think its better to have one term for both.

Reply:
That's what I'm suggesting - one term for both, but Lock height is a pretty accepted industry term for this